drools-wb-screens/drools-wb-globals-editor/drools-wb-globals-editor-client/src/main/java/org/drools/workbench/screens/globals/client/editor/AddGlobalPopupViewImpl.html

```diff
@@ -29,7 +29,7 @@
             </div>
         </form>
         <div class="modal-footer">
-            <button id="cancelButton" data-i18n-key="Cancel" class="btn-cancel"></button>
+            <button id="cancelButton" data-i18n-key="Cancel" class="btn btn-cancel"></button>
```

Comment:
I think we need to change this line to something like this.
```suggestion
            <button id="cancelButton" data-i18n-key="Cancel" class="btn btn-default"></button>
```

See the current result:
<img width="637" alt="Screen Shot 2019-07-29 at 09 20 28" src="https://user-images.githubusercontent.com/1079279/62048124-71b62b00-b1e2-11e9-9568-ffa6495f9985.png">

And see the result with the suggestion above applied:
<img width="648" alt="Screen Shot 2019-07-29 at 09 20 45" src="https://user-images.githubusercontent.com/1079279/62048140-7d095680-b1e2-11e9-90dd-c4d9b9a2f30f.png">